cltk/tests/test_languages/test_old_english.py

```diff
@@ -7,7 +7,7 @@
 from cltk.corpus.swadesh import Swadesh
 from cltk.tag.pos import POSTag
 from cltk.phonology.syllabify import Syllabifier
-from cltk.lemmatize.old_english.lemma import OldEnglishDictioraryLemmatizer
+from cltk.lemmatize.old_english.lemma import OldEnglishDictionaryLemmatizer
```

Comment:
I started at this for 30 seconds looking for the diff. Got it :)

Reply:
Shitty typo, sorry.